op-challenger/game/fault/types/position.go

```diff
@@ -57,6 +57,14 @@ func (p Position) RelativeToAncestorAtDepth(ancestor uint64) (Position, error) {
 	return NewPosition(int(newPosDepth), newIndexAtDepth), nil
 }
 
+// IndexForParentDepth returns the index of a parent position at a given depth.
+func (p Position) IndexForParentDepth(depth int) *big.Int {
```

Comment:
The parent is strictly one level higher than the current position.  So naming-wise I think this would just be `IndexAtDepth` or `IndexAtAncestorDepth`.